src/compiler/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "paths": {
+      "#compiler": ["./"],
+      "#compiler/*": ["./*"]
```

Comment:
Does this subfolder not inherit paths defined in the root?

Reply:
Follow-up: can `tsconfig` be dynamic/JS, so this tsconfig could read whatever aliases are defined above?

Reply:
gotcha, thanks for the clarification